### PR TITLE
Databases_Week2_Exercise

### PR DESCRIPTION
**Pull request by myselfankitaa**

### CODE REVIEW

File: databases/week2/JSONExercise.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+INSERT INTO article(id, title, content) 
+VALUES(1, "BREAKING NEWS: Water is wet!", "Scientists have discovered that water is wet, it's amazing what.... ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur."),
+(2, "Heavy Snowfall Expected this Weekend", "Lots of snow is expected... Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."),
+(3, "BREAKING NEWS: These 10 Clickbait Titles Are Bad for Your Health, Number 7 Will SHOCK You!", "Haha, you clicked! Minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat")
+
+CREATE Table tags(
+  id INT PRIMARY KEY AUTO_INCREMENT,
+  name VARCHAR(20),
+  article_id INT NOT NULL,
```
Comment (NatsGt) [inline databases/week2/JSONExercise.sql:37]: A tag can be in several articles. This means that you would have repeated names in this table, one for each article it appears. This means it is a many to many relation. An article can have many tags, and a tag can appear in many articles.

File: databases/week2/class_n_homeWork_1_3.sql [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+join user_task on user.id = user_task.user_id
+JOIN task ON user_task.task_id= task.id
+join status on task.status_id = status.id
+where status.name = "Done"
+group BY user.name
+
+
+--Home-Work Part-1
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (36, 'DO homeworks', 'Title says it all, just do your homework', '2018-12-17 03:48:18', '2018-12-27 16:48:59','2019-01-20 15:05:39' , 2, 1);
```
Comment (NatsGt) [inline databases/week2/class_n_homeWork_1_3.sql]: As the id is set to autoincrement, you don't need to pass a value. You can just do `INSERT INTO task (title,...) VALUES ('Do homeworks',...)`

**Review by NatsGt — CHANGES_REQUESTED** [review]

File: databases/week2/class_n_homeWork_1_3.sql (hunk 2 above)
Comment (myselfankitaa) [reply databases/week2/class_n_homeWork_1_3.sql]: Ok noted, And I also made the changes in the query accordingly.

File: databases/week2/JSONExercise.sql (hunk 1 above)
Comment (myselfankitaa) [reply databases/week2/JSONExercise.sql:37]: Thanks to point it out. I made the changes in ERD and Data. Also created a junction table between tags and article to establish many-to-many relationship.

**Pull request merged**